src/components/diceManager.js

```diff
@@ -22,6 +48,32 @@ AFRAME.registerComponent("dice-manager", {
       const new_total_dices =
         Number(el.getAttribute("total_dices")) + Number(num_dices);
       el.setAttribute("total_dices", new_total_dices);
+      setTimeout(() => {
+        const diceResults = {
+          1: 0,
+          2: 0,
+          3: 0,
+          4: 0,
+          5: 0,
+          6: 0,
+          total: 0,
+        };
+        const dices = sceneEl.querySelectorAll(`.dice`);
+        dices.forEach((d) => {
+          const { x, z } = d.object3D.rotation;
+          const face = diceManager.getDiceFace(x, z) || 3;
+          diceResults[face] += 1;
+          diceResults["total"] += 1;
+        });
+        el.setAttribute("dice_results", JSON.stringify(diceResults));
+        const dice_results = el.getAttribute("dice_results");
+        el.emit("update-results", { dice_results });
+      }, PROCESS_DICE_DELAY);
```

Comment:
We need to be careful with using `setTimeout`, since when we throw many dices the text is not being updated (cf. asynchronicity).